app/Makefile.version

```diff
@@ -1,3 +1,6 @@
+# This is the `transaction_version` field of `Runtime`
 APPVERSION_M=5
+# This is the `spec_version` field of `Runtime`
 APPVERSION_N=26
+# This is the `impl_version` field of `Runtime`
```

Comment:
This is not correct, we use this for internal patches.

Reply:
Thanks. Fixed.